Makefile

```diff
@@ -78,36 +71,37 @@ docker-stop: # Stop containers for this project
 restart: docker-restart # Alias to docker-restart
 docker-restart: docker-stop docker-start # Restart containers for this project
 
-composer-install: # Installs composer packages from composer.lock file
+composer-install: # Installs Composer packages from composer.lock file
 	$(CURDIR)/bin/composer install \
 		  --ignore-platform-reqs \
 		  --no-interaction \
 		  --no-progress
 
+composer-update: # Update all composer managed libraries
+	$(CURDIR)/bin/composer update \
+		--ignore-platform-reqs -vvv
 
-composer-update: # Update lock file from composer.json and rebuild images
+composer-update-lock:
 	$(CURDIR)/bin/composer update \
-		  --lock \
-		  --no-scripts \
-		  --no-autoloader
+		--lock
 
-drupal-upgrade: # Update drupal core
+drupal-upgrade: # Update Drupal Core
 	$(CURDIR)/bin/composer update drupal/core \
-		  --lock \
-		  --with-all-dependencies
+		--with-all-dependencies \
+		--ignore-platform-reqs
 
-destroy: composer-purge docker-destroy # Take town and remove all data related to this project's current state
+destroy: composer-purge docker-destroy # Take down and remove all data related to this project's current state
 
 docker-destroy:
 	docker-compose -f ${DOCKER_COMPOSE_FILE} down -v
 
 composer-purge:
-	docker exec -i ${PROJECT}-php rm -rf /var/www/webroot/core/*
-	docker exec -i ${PROJECT}-php rm -rf /var/www/webroot/libraries/*
-	docker exec -i ${PROJECT}-php rm -rf /var/www/webroot/modules/contrib/*
-	docker exec -i ${PROJECT}-php rm -rf /var/www/webroot/profiles/contrib/*
-	docker exec -i ${PROJECT}-php rm -rf /var/www/webroot/themes/contrib/*
-	docker exec -i ${PROJECT}-php rm -rf /var/www/vendor/*
+	$(CURDIR)/bin/tool rm -rf /var/www/webroot/core/*
```

Comment:
These commands are close, but we might need to rethink them. They require the containers to be running, which might not always be the case in the event of a `make destroy`. A user might want to purge all their stuff out in the event, for example, that they have an issue with the php container and the container won't stay running. If this is the case, nothing gets deleted and `make destroy` is not completed.

Reply:
Agreed, `bin/tool` doesn't solve that problem yet so we still need something else. Likely something that runs alpine:latest and doesn't rely on any images being built.